InteractiveHtmlBom/core/config.py

```diff
@@ -304,11 +304,12 @@ def safe_set_checked_strings(clb, strings):
         dlg.finish_init()
 
     @classmethod
-    def add_options(cls, parser):
+    def add_options(cls, parser, version):
         # type: (argparse.ArgumentParser) -> None
```

Comment:
Please update the type hint.

Reply:
I look at code in VSCode with pylance language server. I don't know how to run it manually.